katello/katello-restore

```diff
@@ -46,6 +46,7 @@ def restore_databases
     `service postgresql start`
     `runuser - postgres -c "dropdb foreman"`
     `runuser - postgres -c "dropdb candlepin"`
+    `runuser - postgres -c "pg_restore -C -d postgres #{DIR}/pg_globals.dump"`
```

Comment:
I am getting this error: `pg_restore: [archiver] input file appears to be a text format dump. Please use psql.` I don't think we can use pg_restore with pg_dumpall.